scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -191,6 +199,37 @@ def _highly_variable_genes_seurat_v3(
         return df
 
 
+@numba.njit(cache=True, parallel=True)
+def _clip_sparse(
+    indptr: NDArray[np.integer],
+    indices: NDArray[np.integer],
+    data: NDArray[np.floating],
+    *,
+    n_rows: int,
+    n_cols: int,
+    nnz: int,
```

Comment:
n_rows and nnz are unused, should they be removed or is that a mistake?

Reply:
This is not 100%final. I'm still unsure if singlethreaded might be better than multithreaded

Reply:
OK, why would that be? Memory? Locality? I’m just guessing here haha